test/layouts/gate_level_layout.cpp

```diff
@@ -1073,6 +1079,7 @@ TEST_CASE("Move nodes", "[gate-level-layout]")
 
```

Comment:
Could you add a test particularly moving around crossings and check if their tracking behaves correctly?